frontend/src/layouts/JiraDashboard/Table/Table.js

```diff
@@ -0,0 +1,38 @@
+import React, { useCallback } from "react";
+import { Table as Tablebody } from "baseui/table-semantic";
+import { Pagination as TableFooter } from "baseui/pagination";
+import usePagination from "../usePagination.js";
+import Loader from "../../../components/Loader/Loader.js";
+
+const Table = ({ columnTitles, jqlQuery }) => {
+  const { data, pageNumber, totalPages, setPageNumber, loading } =
+    usePagination(jqlQuery);
+
+  const changePage = useCallback(
+    ({ nextPage }) => {
+      setPageNumber(Math.min(Math.max(nextPage, 1), totalPages));
+    },
+    [totalPages]
+  );
+  return (
+    <>
+      {loading === true ? (
+        <Loader />
+      ) : (
+        <div id="table">
```

Comment:
use classname not ids

Reply:
why is this table in JIRADashboard directory? gitlab table should also be rendered through same TableComponent